src/components/StatsDescription/index.tsx

```diff
@@ -0,0 +1,39 @@
+import Typography from '@/components/Typography';
+import styles from './style.module.scss';
+
+const StatsDescription: React.FC = () => {
+  return (
+    <div className={styles.container}>
+      <Typography className={styles.title} variant="display/heavy/small">
+        Take <span className={styles.diamond}>Diamond</span>
+        <span className={styles.hacks}>Hacks</span> to new heights!
```

Comment:
This could potentially be its own component since it's also used in "What is DiamondHacks?" but not important